packages/block-library/src/navigation/index.php

```diff
@@ -161,7 +164,9 @@ private static function get_inner_blocks_html( $attributes, $inner_blocks ) {
 		$is_list_open      = false;
 
 		foreach ( $inner_blocks as $inner_block ) {
-			$is_list_item = in_array( $inner_block->name, static::$nav_blocks_wrapped_in_list_item, true );
+			$inner_block_markup = static::get_markup_for_inner_block( $inner_block );
+			$p                  = new WP_HTML_Tag_Processor( $inner_block_markup );
+			$is_list_item       = $p->next_tag( 'LI' );
```

Comment:
Now that we have the filter we want to drop this auto check, right?

Reply:
Yes that is correct  👍